src/confcom/azext_confcom/tests/latest/test_confcom_arm.py

```diff
@@ -1797,21 +1801,26 @@ class PolicyGeneratingArmInfrastructureSvn(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
+        cls.minimum_svn = "2"
```

Comment:
I think we have this as a constant for this in the config file we can use

Reply:
This test is for using a non-default value for the minimum_svn so I think it should be something other than what's in the config file